ldap/servers/slapd/filter.c

```diff
@@ -1032,6 +1032,52 @@ slapi_filter_get_subfilt(
     return (0);
 }
 
+/* copy/merged from ldmb_attr.c:free_the_filter_bits/replace_filter
+ * it replaces the bits of the Slapi_Filter with the ones taken from strfilter
+ */
+int
+slapi_filter_replace_strfilter(Slapi_Filter *f, char *strfilter)
```

Comment:
should probably move free_the_filter_bits and replace_filter out of ldbm_attr.c here as slapi_ function
 and use them in ldbm_attr ( with a flag: slapi_filter_replace_strfilter(Slapi_Filter *f, char *strfilter, int free_bits)

Reply:
I did move those functions from ldbm_attr to filter.c